.goreleaser.yaml

```
@@ -118,4 +118,4 @@ changelog:
 release:
   prerelease: auto
 git:
-  prerelease_suffix: "-rc"
+  prerelease_suffix: "-*"
```

Based on Discord we can use just hyphen

```suggestion
  prerelease_suffix: "-"
```